server/src/app.module.ts

```diff
@@ -93,6 +95,14 @@ class BaseModule implements OnModuleInit, OnModuleDestroy {
       this.jobRepository.startWorkers();
     }
 
+    this.eventRepository.setAuthFn(async (client) =>
```

Comment:
`EventRepository` can't reference `AuthService` directly anymore, otherwise it causes a circular dependency, so I moved it here and hooked it up via a callback.

`AuthService` => `BaseService` => `EventRepository`.